packages/vike-react-chakra/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "vike-react-chakra",
+    "version": "0.0.1",
```

Comment:
I suggest we go ahead and directly release `1.0.0`.

Reply:
Noted 👍